jsig-allow.json

```diff
@@ -0,0 +1,5 @@
+{
+    "allow": [
+        {"jsig":"0.1.5","path":"node_modules/jsinstallguard","v":"0.1.5","cmd":"cp ./{.npmrc,.yarnrc,jsig-allow.json} ../../"}
```

Comment:
I wonder if we add this trust within the code itself (maybe just add it to the allow array in the code). That way if someone has an existing jsig-allow files then we can just leave it alone.

Reply:
Yeah, I agree, this was knocked up quickly for POC.